app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -99,11 +108,13 @@
       <ImageView
         android:layout_width="36dp"
         android:layout_height="36dp"
+        android:layout_marginTop="-2dp"
         android:src="@{viewModel.showPassword ? @drawable/ic_hide_eye_icon : @drawable/ic_show_eye_icon}"/>
       <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="-2dp"
+        android:fontFamily="sans-serif"
```

Comment:
The color code of this text view in mocks in `#008098`

Reply:
Changed textcolor to #008098.